src/Concerns/ManagesInvoices.php

```diff
@@ -131,7 +131,7 @@ public function findInvoice($id)
             $stripeInvoice = StripeInvoice::retrieve(
                 $id, $this->stripeOptions()
             );
-        } catch (Exception $exception) {
+        } catch (StripeInvalidRequestException $exception) {
```

Comment:
This was apparently referencing an invalid exception. Seems like no-one has hit this snag so far...